packages/meditrak-server/src/routes/accessRequests/GETAccessRequests.js

```diff
@@ -3,16 +3,18 @@
  * Copyright (c) 2017 - 2020 Beyond Essential Systems Pty Ltd
  */
 
+import { QUERY_CONJUNCTIONS } from '@tupaia/database';
 import { GETHandler } from '../GETHandler';
 import {
   assertAnyPermissions,
   assertBESAdminAccess,
   assertTupaiaAdminPanelAccess,
+  hasBESAdminAccess,
 } from '../../permissions';
-import {
-  assertAccessRequestPermissions,
-  filterAccessRequestsByPermissions,
-} from './assertAccessRequestPermissions';
+import { TUPAIA_ADMIN_PANEL_PERMISSION_GROUP } from '../../permissions/constants';
```

Comment:
Instead of 'reaching in', we could export the permissions constants from the root of the permissions folder

Reply:
Looks like this was actually already the case, I was just following a pattern from an outdated file.